kafka-scheduler/src/test/java/com/mesosphere/dcos/kafka/scheduler/RecoverySchedulerTest.java

```diff
@@ -85,14 +88,22 @@ public void testReplaceNonLastBroker() throws Exception {
                 replaceTaskInfo,
                 getDummyBrokerTaskInfo(1),
                 getDummyBrokerTaskInfo(2));
+        List<Protos.TaskStatus> statuses=getTestKafkaStatuses(replaceTaskInfo,taskInfos);
         when(stateStore.fetchTasks()).thenReturn(taskInfos);
-        when(stateStore.fetchTasksNeedingRecovery()).thenReturn(Arrays.asList(replaceTaskInfo));
+        when(stateStore.fetchStatuses()).thenReturn(statuses);
+        when(stateStore.fetchTask(any())).thenReturn(Optional.of(replaceTaskInfo));
+        when(stateStore.fetchStatus(any())).thenReturn(Optional.of(statuses.get(0)));
 
-        DefaultRecoveryScheduler recoveryScheduler = getTestKafkaRecoveryScheduler();
-        List<Protos.OfferID> acceptedOfferIds = recoveryScheduler.resourceOffers(
+        DefaultRecoveryPlanManager recoveryPlanManager = getTestKafkaRecoveryPlanManager();
+        DefaultPlanScheduler planScheduler = getTestKafkaScheduler();
+
+        List<Protos.OfferID> acceptedOfferIds = new ArrayList<>(
+                planScheduler.resourceOffers(
                 driver,
```

Comment:
Move these over a level of indentation so it's apparent that they're arguments to `planScheduler.resourceOffers`.